src/Gui/DlgGeneral.ui

```diff
@@ -507,6 +507,125 @@ this according to your screen size or personal taste</string>
          </item>
         </layout>
        </widget>
+      </item>
+	  <item>
+       <widget class="QGroupBox" name="GroupBox4">
+        <property name="title">
+         <string>Show Workbench selection widget in </string>
+        </property>
+        <layout class="QVBoxLayout">
+         <property name="spacing">
+          <number>6</number>
+         </property>
+         <property name="leftMargin">
+          <number>11</number>
+         </property>
+         <property name="topMargin">
+          <number>11</number>
+         </property>
+         <property name="rightMargin">
+          <number>11</number>
+         </property>
+         <property name="bottomMargin">
+          <number>11</number>
+         </property>
+         <item>
+          <layout class="QHBoxLayout">
+           <property name="spacing">
+            <number>6</number>
+           </property>
+           <property name="leftMargin">
+            <number>0</number>
+           </property>
+           <property name="topMargin">
+            <number>0</number>
+           </property>
+           <property name="rightMargin">
+            <number>0</number>
+           </property>
+           <property name="bottomMargin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="Gui::PrefCheckBox" name="WBSLeftCorner">
+             <property name="toolTip">
+              <string>Show the workbench selector as a menu bar left corner widget</string>
+             </property>
+             <property name="text">
+              <string>Left corner</string>
+             </property>
+             <property name="checked">
+              <bool>false</bool>
+             </property>
+             <property name="prefEntry" stdset="0">
+              <cstring>WBSLeftCorner</cstring>
+             </property>
+             <property name="prefPath" stdset="0">
+              <cstring>General</cstring>
+             </property>
+            </widget>
+           </item>
```

Comment:
Does really one would activate all possible locations ? IMO it would make sense to use `PrefRadioButton` instead.